docs/docs/tutorial/part-2/index.mdx

```diff
@@ -233,9 +235,30 @@ const AboutPage = () => {
 export default AboutPage
 ```
 
-2. In a web browser, visit `localhost:8000/about`. When your development server finishes rebuilding your site, the About page should look something like this:
+2. Add a page title to your page. Gatsby lets you define a `<title>` and other [document metadata](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head) with the [Gatsby Head API](/docs/reference/built-in-components/gatsby-head/). You have to export a component called `Head` from your page to apply the metadata.
+Adding such metadata helps search engines like Google to better understand your site. For this tutorial we'll only be adding a title to this page but you can also later add other metadata.
+
+```js:title=src/pages/about.js
+import * as React from 'react'
+
+const AboutPage = () => {
+  return (
+    <main>
+      <h1>About Me</h1>
+      <p>Hi there! I'm the proud creator of this site, which I built with Gatsby.</p>
+    </main>
+  )
+}
+
+// Yay! We defined a page title. This shows up on the browser tab
+export const Head = () => <title>About Me<title/>
+
+export default AboutPage
+```
```

Comment:
Add a "Key Gatsby Concept" below this second point here about the Head API

Reply:
👍